### PR TITLE
source files that makes the project website

### PR DESCRIPTION
**Pull request by sreev**
initial commit to new branch 'site-src'.

### CODE REVIEW

File: content/_data/contributors.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+- name: Josh Clemm
+  apacheId:
+  githubId:
+  role: Committer
+  org:
+  homepage: 
+
+
+- name: Caleb Meier
```
Comment (joshelser) [inline content/_data/contributors.yml:69]: copy/paste error? I don't think Caleb is involved with Gossip.

**Comment by joshelser** [comment]
Will try to build this locally today to poke around.

It seems like you have some images copied from Zeppelin that you didn't remove? Want to rip those out?

**Comment by joshelser** [comment]
Why the parent "content" directory? I think this directory is unnecessary.

**Comment by joshelser** [comment]
I was able to build the website, but the navigation links are messed up. They always construct a URL off of the current page. For some reason, it seems like {{site.baseurl}} isn't working properly. Have you observed this?

Edit: Oh, you just need to set `baseurl: /` in _config.yml and then things work fine.

**Comment by joshelser** [comment]
I'm happy with the content WRT ASF incubating branding expectations. @edwardcapriolo how do you feel about the content from a "product" perspective?

**Comment by edwardcapriolo** [comment]
This is good. I can add/edit as need be (the first thing I want to tackle is our merge commit review policy). I will have more ideas one I can see it I am sure.

**Comment by joshelser** [comment]
Let me just fix these two things and merge it in.

**Comment by sreev** [comment]
done. thank you, josh.

**Pull request closed without merging**